packages/@react-spectrum/table/stories/TableDnD.stories.tsx

```diff
@@ -119,6 +119,15 @@ export const DragBetweenTablesRootOnly: TableStory = {
   name: 'Drag between tables (Root only)'
 };
 
+export const DragBetweenTablesRootOnlyNoDisabled: TableStory = {
+  render: (args) => (
+    <Flex direction="row" wrap alignItems="center">
+      <DragBetweenTablesRootOnlyExample tableViewProps={args} />
```

Comment:
If you select all, start a drag, and drop the items in the other table, emptying out the table where the drag started, the select all checkbox in the header of that initial table remains checkIed. Bug?